exercise_150514_Hangman.rb

```diff
@@ -0,0 +1,124 @@
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @wrong_count = 0
+    @ascii = [
+      "|     _________".light_black,
+      "|     |/      |".light_black,
+      "|     |".light_black,
+      "|     |".light_black,
+      "|     |".light_black,
+      "|     |".light_black,
+      "|     |".light_black
+      ]
+     @man_array = [
+      "|     |      / \\".colorize(:red),
+      "|     |       |".colorize(:cyan),
+      "|     |      \\|/".colorize(:yellow),
+      "|     |      (_)".colorize(:magenta)
+    ]
+    @user_guesses = []
+
+    @word = pick_word
+    @underlines = create_underlines
+    puts "Answer: #{@word.join.upcase}" ##TODO: Remove this before it goes to the user.
+
+    @play_status = true
+    play
+  end
+
+  def pick_word
+    word_options = %w(scared selection shake shaking shallow shout silly simplest slight slip slope soap solar species spin stiff swung tales thumb tobacco toy trap treated tune university vapor vessels wealth wolf zoo)
+    word = word_options.sample.split(//)
+    return word
+  end
+
+  def create_underlines
+    underlines = ["| ____|___ word: ".light_black]
+    @word.length.times do
+      underlines.push("_ ")
+    end
+
+    return underlines
+  end
+
+  def ascii_art
+    @ascii[7] = @underlines.join
+    puts @ascii
+  end
+
+  def play
+    puts "Welcome! Let's play Hangman!".cyan
+    while true
+      ascii_art
+
+      puts "You already guessed: #{@user_guesses.join(" ")}"
```

Comment:
We talked about having it not put this line prior to their first guess, but decided we didn't have time to complete it before the assignment was due ;)